### PR TITLE
Lots of new documentation for running on other systems

### PR DESCRIPTION
**Pull request by ewels**
Also tidied the headers of the config files.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-`SciLifeLab/NGI-RNAseq` is specified as the pipeline name.
+1. [Install Nextflow](#install-nextflow)
+2. [Install the pipeline](#install-the-pipeline)
+3. Configure the pipeline
+    * [Swedish UPPMAX System](#31-configuration-uppmax)
+    * [Other Clusters](#32-configuration-other-clusters)
+    * [Docker](#33-configuration-docker)
+    * [Amazon AWS](#34-configuration-amazon-ec2)
+
+## 1) Install NextFlow
```
Comment (Hammarn) [inline docs/installation.md]: How about having something like this here;
https://github.com/IARCbioinfo/needlestack/blob/master/README.md#quick-start
```
curl -fsSL get.nextflow.io | bash
And move it to a location in your $PATH (/usr/local/bin for example here):

sudo mv nextflow /usr/local/bin
```

File: main.nf [hunk 2]
```diff
@@ -361,7 +361,8 @@ process fastqc {
  */
 process trim_galore {
     tag "$name"
-    publishDir "${params.outdir}/trim_galore", mode: 'copy'
+    publishDir "${params.outdir}/trim_galore", mode: 'copy',
```
Comment (Hammarn) [inline main.nf]: What will this do?
It changes the filename somehow, but to what?

Comment (ewels) [inline main.nf]: If the filename contains the string `_fastqc` (the index position of that substring is > 0), the filter is applied which prepends `FastQC/` to the filename.

In other words, it puts the FastQC reports into a subdirectory.

File: docs/installation.md (hunk 1 above)
Comment (ewels) [inline docs/installation.md]: Good idea, will do 👍

**Comment by Hammarn** [comment]
Nicely done!

File: main.nf (hunk 2 above)
Comment (ewels) [inline main.nf]: I stole the idea from @MaxUlysse ;)

Comment (MaxUlysse) [inline main.nf]: I did that ?

Comment (Hammarn) [inline main.nf]: Yeah, you mentioned that in the #nextflow channel on slack

Comment (MaxUlysse) [inline main.nf]: Ah yes, I remember

**Pull request merged**